pkg/pillar/hypervisor/kvm.go

```diff
@@ -387,12 +387,19 @@ const qemuRootPortPciPassthruTemplate = `
   addr = "{{printf "0x%x" .PCIId}}"
 `
 
+const qemuPCIPassthruBridgeTemplate = `
+[device "pcie-bridge.{{.PCIId}}"]
+  driver = "pcie-pci-bridge"
```

Comment:
why pcir-pci and no pcie-pcie? Shown PCIe device ans PCI has consequences and usually not a good idea

Reply:
`pcie-pcie` does not exist according to https://github.com/qemu/qemu/blob/master/docs/pcie.txt#L39

Reply:
sure, because there is no need to translate. You either create a dedicated port to control BDF assignments or plug device into a PCIe switch